src/shared/types/input.ts

```diff
@@ -2,4 +2,8 @@ import React from "react";
 
 export interface MAButtonTypes {
   children: React.ReactNode;
+  fill: boolean;
```

Comment:
```suggestion
  varient?: "outlined" | "contained";
```
fill도 방법이 될 수 있기는 한데 이런식으로 하는 걸 연습해보도록 합시다.

그리고 얘도 optional로 줘서 props를 주지 않아도 기본 세팅이 contained로 되어있도록 만드는게 
좋아보입니다.

MUI에서도 이런 방식을 쓰고 있어요
[MUI Button](https://mui.com/components/buttons/)